server/startup/migrations/v238.ts

```diff
@@ -0,0 +1,28 @@
+import { Mongo } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations/server';
+import { Settings } from '../../../app/models/server/raw';
+import { settings } from '../../../app/settings/server';
+
+Migrations.add({
+	version: 238,
+	up() {
+		const query = {
+			_id: { $in: [/^Accounts_OAuth_Custom-?([^-_]+)$/] },
+			value: true,
+		};
+
+		const isCustomOAuthEnabled = !!Promise.await(Settings.findOne(query));
+		const LDAPEnabled = settings.get('LDAP_Enable');
+		const SAMLEnabled = settings.get('SAML_Custom_Default');
+
+		if (isCustomOAuthEnabled || LDAPEnabled || SAMLEnabled) {
+			return;
+		}
```

Comment:
Now that I understood this is updating an existing banner (if it exists) I don't believe we need these checks. The update will update only if the banner already exists